jupyterhub/metrics.py

```diff
@@ -21,13 +21,13 @@
 from prometheus_client import Histogram
 
 REQUEST_DURATION_SECONDS = Histogram(
-    'request_duration_seconds',
+    'jupyter_request_duration_seconds',
```

Comment:
jupyterhub_ as a prefix instead of jupyter_ maybe? Jupyter server also will have metrics